docs/organization/membership/index.mdx

```diff
@@ -49,7 +49,8 @@ Organization-level roles enable broad access to the entire organization, as desc
 | Can transfer projects to another organization                                   |             |            |           |             | ✓         |
 | Can remove an organization                                                      |             |            |           |             | ✓         |
 
-\*For Business and Enterprise plans, even Org Members who are not a Team Admin for any team can create a new personal project. When creating this project, the member will get assigned a new personal team that they will become a Team Admin for. [Team Admin permissions](#permissions-1) will then apply for this team.
+\*Org Members can only edit the project ownership rule setting.
+\*\*For Business and Enterprise plans, even Org Members who are not a Team Admin for any team can create a new personal project. When creating this project, the member will get assigned a new personal team that they will become a Team Admin for. [Team Admin permissions](#permissions-1) will then apply for this team.
```

Comment:
```suggestion
\*Org Members can only edit the project ownership rule setting.

\*\*If your organization is on a Business or Enterprise plan, anyone who's part of the org will be able to create a project without having to be a Team Admin. When org members create a project, they'll become the Team Admin for that project by default and have all the permissions of the [Team Admin](#permissions-1) for that project.
```
@jangjodi I'm not sure I fully understand the difference between being able to add/remove teams versus only being able to edit the rules. Could you clarify? :)

Reply:
That was actually a typo, thank you for catching @lizokm! I mean to add the check mark in the "Can change project settings" column